server/controllers/utils/overviewController/buildOverviewActions.ts

```diff
@@ -41,7 +41,7 @@ export default (
     })
   }
 
-  if (permissions.keyWorker?.edit) {
+  if (user.authSource === 'nomis' && user.keyWorkerAtPrisons?.[prisonerData.prisonId]) {
```

Comment:
very minor point - Do we need to check the auth source? This check is essentially covered by the key worker check